### PR TITLE
Start working on the glossary.

### PR DESCRIPTION
**Pull request by geeknoid**
- Also fixed a few things in the installation guide

- Update a few other docs along the way.

Preview site here: https://geeknoid.github.io/istio.github.io/true-index.html

### CODE REVIEW

File: _docs/tasks/installing-istio.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    ```bash
+    kubectl apply -f ./kubernetes/addons/grafana.yaml
+    kubectl apply -f ./kubernetes/addons/prometheus.yaml
+    kubectl apply -f ./kubernetes/addons/servicegraph.yaml
+    ```
+
+    The Grafana image provided as part of this sample contains a built-in Istio dashboard that you can access from:
+
+    ```bash
+    http://<grafana-svc-external-IP>:3000/dashboard/db/istio-dashboard
```
Comment (esnible) [inline _docs/tasks/installing-istio.md]: These instructions expect the user to know the grafana-svc-external-IP which has not been introduced yet. They expect the user to understand how to change the Service from type LoadBalancer to type NodePort and under what circumstances that is needed. They expect the user to know how to run _kubectl proxy_ and how to edit Grafana's Istio-dashboard.

Comment (geeknoid) [inline _docs/tasks/installing-istio.md]: @andraxylia Andra, can you address this in a follow-on PR?

Comment (andraxylia) [inline _docs/tasks/installing-istio.md]: Yes, I will.

File: _docs/reference/glossary.md [hunk 2]
```diff
@@ -8,35 +8,8 @@ bodyclass: docs
...
-                    <li class="submenu">
-                        <h6 class="arrow-r">Microservice</h6>
-                        <div class="submenu-content">
-                            <p>TBD</p>
-                        </div>
-                    </li>
-                    
-<!-- Words to add to the glossary
+
+<!-- Ed's ideas for words to add to the glossary
```
Comment (andraxylia) [inline _docs/reference/glossary.md]: Remove reference to Ed.

**Pull request merged**